recirq/quantum_chess/quantum_board.py

```diff
@@ -343,6 +345,35 @@ def get_probability_distribution(self,
 
         return self.probabilities
 
+    def _generate_board_accumulations(self, repetitions: int = 1000) -> None:
+        """ Samples the state and generates the accumulated
+        probabilities of the whole board.
+        """
+        self.board_probabilities = {}
+
+        samples = self.sample(repetitions)
+        for sample in samples:
+            if sample not in self.board_probabilities:
+                self.board_probabilities[sample] = 1
+            else:
+                self.board_probabilities[sample] += 1
+
+        for board in self.board_probabilities:
+            self.board_probabilities[board] = float(self.board_probabilities[board]) / float(repetitions)
+
+        self.board_accumulations_repetitions = repetitions
+
+    def get_board_probability_distribution(self,
+                                           repetitions: int = 1000) -> Dict[int, float]:
+        """Returns the probability of the whole board being in different states.
```

Comment:
Language is confusing again... Maybe "Returns a probability distribution for each board found in the sample"?

Reply:
Thanks Chris. It's changed.